frontend/src/metabase/embedding/embedding-hub/utils/embedding-hub-steps.ts

```diff
@@ -52,18 +52,11 @@ export const getEmbeddingHubSteps = (): EmbeddingHubStep[] => {
     },
     actions: [
       {
-        title: t`Generate a dashboard`,
+        title: t`Create a dashboard`,
         description: t`Automatically generate a dashboard from your data using x-rays.`,
         modal: { type: "xray-dashboard" },
         variant: "outline",
       },
-      // {
-      //   title: t`Build a dashboard`,
-      //   description: t`Build your own dashboard from scratch.`,
-      //   modal: { type: "new-dashboard" },
-      //   variant: "subtle",
-      //   optional: true,
-      // },
       {
         title: t`Create models`,
```

Comment:
Just notice we use plural here. We only instruct users to "do" 1 thing e.g. creating a dashboard, connect to a db, but why we want to let them creating multiple models not just one?